modules/identity/src/Volo.Abp.Identity.Web/Pages/Identity/Users/CreateModal.cshtml.cs

```diff
@@ -28,9 +28,20 @@ public async Task OnGetAsync()
         {
             UserInfo = new UserInfoViewModel();
 
+            var roleDtoList = await _identityRoleAppService.GetAllListAsync();
```

Comment:
`_identityRoleAppService.GetAllListAsync` is already called in the next line. Don't duplicate!